SalesforceSDK/Core/Auth/AccountManager.cs

```diff
@@ -67,9 +67,6 @@ public static void DeleteAccount()
         {
             Account account = GetAccount();
             AuthStorageHelper.DeletePersistedCredentials(account.UserName, account.UserId);
-
-            // raise event about account logout
-            AuthenticatedAccountChanged?.Invoke(new AuthenticatedAccountChangedEventArgs(account, null));
```

Comment:
the line right above this `AuthStorageHelper.DeletePersistedCredentials`, that's where the other change is that causes the property to get set which raises the event